addons/languages/rusyn/pack/build.gradle

```diff
@@ -0,0 +1,6 @@
+ext.status_icon_text = "rue"
+
+ext.dictionaryInputPossibleCharacters = null//"".toCharArray()
+ext.dictionaryInputAdditionalInnerCharacters = null//"".toCharArray()
+
+apply from: "$rootDir/gradle/language_pack_lib.gradle"
```

Comment:
this should be `apply from: "$rootDir/addons/gradle/language_pack_lib.gradle"`

Reply:
Fixed in commit bf9abef Resolved relative paths and unique id